deployment/v3/utils/prop_migrator/Overview.md

```diff
@@ -137,13 +137,71 @@ This section lists the property files that were updated during the process. It i
+
+#### File Format :-
+The file has the following columns:
+
+* Property file name: The name of the property file where the property is defined.
+* Key: The key (or property name) that identifies the property.
+* Old Value: The value of the property in the old version of config.
+* Latest Value: The value of the property in the latest version of config.
+* Configuration Action: A comment indicating the required manual action for the property.
+  
+#### Comments Used in Configuration Action:
+The "Configuration Action" column includes comments that guide developers on how to handle specific combinations of properties during updates. The following comments are used in the file:
+
+#### 1. update-if-old-value-takes-priority:
+
+Description:
+This comment suggests that in the specific combination of properties, the value from the old version of config should take priority over the value in the latest version. In other words, the old value is considered more important or appropriate for the current context.
+
+Action:
+Developers should manually update the property file in the latest version and set the property value to the old value indicated in the "Old Value" column of the CSV file. This will ensure that the desired configuration from the old version is preserved in the updated config branch.
+
+Use Case:
+This comment is useful when a property in the latest version has been changed, but the older value is still relevant for compatibility, stability, or other reasons. By updating the property file manually, developers can maintain expected behavior in the updated version.
+
+#### 2. copy-property-if-required-in-lts:
+
+Description:
+This comment indicates that a specific property is required in the latest(LTS) version of the config branch, but it is missing from the latest version.
+
+Action:
+Developers should manually add the property along with its old value to the property file in the latest version of the config branch. This ensures that the required property is present in the LTS version, even if it was not part of the latest version's configuration.
+
+Use Case:
+Some properties might be essential for the LTS version due to backward compatibility or other critical reasons. The comment helps developers identify and copy such properties from the old version to the latest version, ensuring their availability in the latest config branch.
+
+#### 4. update-if-value-needs-change:
+
+Description:
+This comment advises developers to update the property's value in the latest version if a change is necessary. It implies that the latest value may not be suitable, and developers should determine the appropriate value for the property.
```

Comment:
this is pointed towards a user, not developers

Reply:
Updated.